panther_manager/include/panther_manager/plugins/action/shutdown_single_host_node.hpp

```diff
@@ -38,14 +38,17 @@ class ShutdownSingleHost : public ShutdownHosts
   static BT::PortsList providedPorts()
   {
     return {
-      BT::InputPort<std::string>("ip", "ip of the host to shutdown"),
-      BT::InputPort<std::string>("username", "user to log into while executing shutdown command"),
-      BT::InputPort<unsigned>("port", "SSH communication port"),
-      BT::InputPort<std::string>("command", "command to execute on shutdown"),
-      BT::InputPort<float>("timeout", "time in seconds to wait for host to shutdown"),
+      BT::InputPort<std::string>("ip", "IP address of the host to shut down."),
+      BT::InputPort<std::string>(
+        "username", "Username to use for logging in and executing the shutdown command."),
+      BT::InputPort<unsigned>("port", "SSH port used for communication (default is usually 22)."),
+      BT::InputPort<std::string>("command", "Shutdown command to execute on the remote host."),
```

Comment:
```suggestion
      BT::InputPort<std::string>("command", "A command to execute on the remote host."),
```

It can be any command.

Reply:
Yes, but if you don't type shutdown command, behaviour of this tree will not be reasonable.